srv/github/pull_package.js

```diff
@@ -56,7 +56,11 @@ module.exports = function make_pull_package() {
 
     // Might be new, wait for npm pull
     if(null == npment) {
-      await new Promise((r)=>setTimeout(r,555))
+      /* QUESTION: Can we put the repository url in nodezoo/orig instead of
+       * adding a dependency on time here?
+       */
+      await sleep(2e3)
+
```

Comment:
QUESTION: Can we put the repository url in nodezoo/orig instead of adding a dependency on time here?